templates/field--node--su-news-topics.html.twig

```diff
@@ -1,45 +1,34 @@
-{# Create classes array #}
-{% set classes = []
-%}
-
-{# BEM inspired class syntax: https://en.bem.info/
-   Enable this code if you would like field classes like "article__tags", where article is the content type and field_tags is the field name.
-{% set classes = classes|merge([
-  bundle ~ '__' ~ field_name|replace({'field_' : ''})|clean_class
-]) %}
-#}
-
-{% set attributes = attributes.addClass(classes) %}
```

Comment:
@boznik @jenbreese none of this logic was doing anything so I removed it all. Please confirm.